ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>I am not sure if it is a guideline requirement, but many notebooks adopt placing a space between the "!" and the command so its clear you executed a system command and not Python code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>